Sources/XcodeGenKit/SchemeGenerator.swift

```diff
@@ -142,12 +142,32 @@ public class SchemeGenerator {
             return XCScheme.BuildAction.Entry(buildableReference: buildableReference, buildFor: buildTarget.buildTypes)
         }
 
+        func getBuildEntryForTestTarget(_ buildTarget: Scheme.BuildTarget) throws -> XCScheme.BuildAction.Entry {
+            
+            // Need to check local Swift Package test case
+            func getBuildableReferenceForSPM(_ target: TargetReference) -> XCScheme.BuildableReference? {
+                if case .project(let project) = target.location,
+                   let package = self.project.getPackage(project), case .local(let path) = package {
```

Comment:
How about break line?

```suggestion
                   let package = self.project.getPackage(project), 
                   case .local(let path) = package {
```

Reply:
Cool :))